GradeCenter.Data/Models/Grade.cs

```diff
@@ -0,0 +1,19 @@
+﻿using GradeCenter.Data.Models.Account;
+using System.ComponentModel.DataAnnotations;
+
+namespace GradeCenter.Data.Models
+{
+    public class Grade
+    {
+        [Key]
+        public Guid Id { get; set; }
+        [Range(2, 6)]
+        public int Number { get; set; }
```

Comment:
Here is not clear what is meant by number. Better name would be "Value" or "Rate".